### ISSUE TITLE
-su: option requires an argument -- b

### ISSUE THREAD

**Issue by rainmanh** [Jumpscale/bash#38]
root@GS9-Bash:~# ZInstall_portal9

```
[+] docker from image jumpscale/js9_full is active, access it through 'ZSSH'
[+] install AYS9
[+] loading or updating AYS source code (branch:)
-su: option requires an argument -- b
Usage: ZCodeGet [-r reponame] [-g giturl] [-a account] [-b branch]
   -r reponame: name or repo which is being downloaded
   -b branchname: defaults to master
   -h: help

check's out jumpscale repo to /opt/code/github/jumpscale/ays9
branchname can optionally be specified.

if specified but repo exists then a pull will be done & branch will be ignored !!!

if reponame not specified then will checkout
- bash
- lib9
- core9
- ays9
- prefab

[+] docker from image jumpscale/ays9 is not active, will try to start
[+] start docker jumpscale/ays9 -> portal9 (port:2222)

**** ERRORLOG ****
FUNCTION: ZCodeGet
mkdir: /opt/code/github/jumpscale
pushd to: /opt/code/github/jumpscale
/opt/code/github/jumpscale ~ /opt/code/github/jumpscale/bash
pushd to: /opt/code/github/jumpscale/prefab9
/opt/code/github/jumpscale/prefab9 /opt/code/github/jumpscale ~ /opt/code/github/jumpscale/bash
git pull
Already up-to-date.
FUNCTION: container
FUNCTION: ZDockerConfig

mkdir: /root/js9host//.cache/pip
mkdir: /root/js9host//cfg
FUNCTION: container
FUNCTION: ZDockerConfig

mkdir: /root/js9host//.cache/pip
mkdir: /root/js9host//cfg
FUNCTION: container
FUNCTION: ZDockerConfig

mkdir: /root/js9host//.cache/pip
mkdir: /root/js9host//cfg
Obtaining file:///opt/code/github/jumpscale/core9
Requirement already satisfied: GitPython>=2.1.3 in /usr/local/lib/python3.5/dist-packages (from JumpScale9==9.1.2)
Requirement already satisfied: click>=6.7 in /usr/local/lib/python3.5/dist-packages (from JumpScale9==9.1.2)
Collecting colored_traceback (from JumpScale9==9.1.2)
Collecting colorlog>=2.10.0 (from JumpScale9==9.1.2)
  Using cached colorlog-3.1.0-py2.py3-none-any.whl
Collecting httplib2>=0.10.3 (from JumpScale9==9.1.2)
Collecting ipython>=6.0.0 (from JumpScale9==9.1.2)
  Using cached ipython-6.2.1-py3-none-any.whl
Requirement already satisfied: libtmux>=0.7.1 in /usr/local/lib/python3.5/dist-packages (from JumpScale9==9.1.2)
Collecting netaddr>=0.7.19 (from JumpScale9==9.1.2)
  Using cached netaddr-0.7.19-py2.py3-none-any.whl
Collecting path.py>=10.3.1 (from JumpScale9==9.1.2)
  Using cached path.py-10.4-py2.py3-none-any.whl
Collecting pystache>=0.5.4 (from JumpScale9==9.1.2)
```

**Comment by rainmanh** [comment]
```
mkdir: /root/js9host//.cache/pip
mkdir: /root/js9host//cfg


Unable to find image 'jumpscale/ays9:latest' locally
latest: Pulling from jumpscale/ays9
22ecafbbcc4a: Already exists
580435e0a086: Already exists
8321ffd10031: Already exists
08b8f28a13c2: Already exists
2b401702069a: Already exists
a3ed95caeb02: Already exists
eae027dcdc0e: Already exists
93bc98227159: Already exists
a3ed95caeb02: Already exists
a3ed95caeb02: Already exists
a3ed95caeb02: Already exists
a3ed95caeb02: Already exists
a3ed95caeb02: Already exists
57f874b22b31: Pulling fs layer
f71312555ea9: Pulling fs layer
a71269776dd4: Pulling fs layer
50a110703842: Pulling fs layer
d58ff31c449d: Pulling fs layer
50a110703842: Waiting
d58ff31c449d: Waiting
f71312555ea9: Verifying Checksum
f71312555ea9: Download complete
a71269776dd4: Verifying Checksum
a71269776dd4: Download complete
d58ff31c449d: Verifying Checksum
d58ff31c449d: Download complete
57f874b22b31: Verifying Checksum
57f874b22b31: Download complete
57f874b22b31: Pull complete
50a110703842: Verifying Checksum
50a110703842: Download complete
f71312555ea9: Pull complete
a71269776dd4: Pull complete
50a110703842: Pull complete
d58ff31c449d: Pull complete
Digest: sha256:b6b913cb049b4b56167787d620c64cdbf410b2d004792f2cf67bd15f49e190d8
Status: Downloaded newer image for jumpscale/ays9:latest
6abf0e118bfaf8a04499ce65f20b747f3504c894cd153ba73b23efdf489ee515
docker: Error response from daemon: driver failed programming external connectivity on endpoint portal9 (6c2bd61c83ac34869f567206d49851cb6d2a51cf3eb54e8200c6f8291d35e14f): Bind for 0.0.0.0:2222 failed: port is already allocated.
```
```
root@GS9-Bash:~# docker ps
CONTAINER ID        IMAGE                COMMAND             CREATED             STATUS              PORTS                  NAMES
d871200c468c        jumpscale/js9_full   "/sbin/my_init"     4 minutes ago       Up 4 minutes        0.0.0.0:2222->22/tcp   ays9
root@GS9-Bash:~# 
```

**Comment by rkhamis** [comment]
Fixed in #40
